bangazonapi/views/product.py

```diff
@@ -102,7 +103,11 @@ def create(self, request):
 
             new_product.image_path = data
 
-        new_product.save()
+        try:
+            new_product.full_clean()
```

Comment:
I believe `clean_fields()` is all that's needed here, not a `full_clean()`, which does more validation than what's required here.

Reply:
update the line code 107 from full_clean to clean_fields as request by Mitchell B. and because the full clean does more validation than what's required here!